chemistry-opencmis-test-tck/pom.xml

```diff
@@ -1,125 +1,142 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Revert changes on this file.

Reply:
Updated the pom with latest dependencies
https://github.com/PogeyanOSS/copper-cms/pull/131/commits/437b67d952a291f2e2bf45bf39441a909a52c805